pages/about.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../css/common.css">
+    <link rel="stylesheet" type="text/css" media="screen and (max-width: 746px)" href="../css/hamburger.css">
+    <link rel="stylesheet" type="text/css" media="screen and (max-width: 746px)" href="../css/mobile.css">
+    <link rel="stylesheet" type="text/css" media="screen and (min-width: 747px)" href="../css/desktop.css">
+    <link rel="stylesheet" type="text/css" media="screen and (min-width: 747px)" href="../css/abtDesktop.css">
+    <link rel="stylesheet" type="text/css" media="screen and (max-width: 746px)" href="../css/abtMobile.css">
+    <title>Capstone About</title>
+</head>
+<body>
+    <header>
+        <nav class="dk top-nav-container">
+            <article class="top-nav">
+                <ul class="social-icon">
+                    <li><img src="../assets/social-icon/facebook_f_64px.png" alt="facebook"></li>
+                    <li><img src="../assets/social-icon/twitter_64px.png" alt="twitter"></li>
+                </ul>
+                <ul class="h-link">
+                    <li><a href="#">English</a></li>
+                    <li><a href="#">My Page</a></li>
+                    <li><a href="#">Logout</a></li>
+                </ul>
+            </article>
+        </nav>
+        <nav class="main-nav">
+            <section class="mobile-menu mb">
+                <label class="hamburger-menu mb" for="menu-checkbox">
+                    <input type="checkbox" id="menu-checkbox" />
+                </label>
+                <aside class="sidebar">
+                    <a class="new-stuff" href="#">Logout</a>
+                    <a href="../index.html">Home</a>
+                    <a href="#">Program</a>
+                    <a href="#">Join</a>
+                    <a href="#">Sponsor</a>
+                    <a href="#">News</a>
+                    <a class="CC" href="#">CC Campaign</a>
+                </aside>
+            </section>
+            <a href="../index.html" class="logo dk">BLK-NG</a>
+            <article class="main-nav-link dk">
+                <a href="../index.html">Home</a>
+                <a href="#">Program</a>
+                <a href="#">Join</a>
+                <a href="#">Sponsor</a>
+                <a href="#">News</a>
+                <a href="#">CC Campaign</a>
+            </article>
+        </nav>
+        <section class="headline-container-abt">
+            <section class="headline-abt">
+                <h2 class="grtings mb">"Hello Sharing world"</h2>
+                <h3 class="title-abt">
+                    CREATIVE COMMONS<br>GLOBAL SUMMIT 2023
+                </h3>
```

Comment:
- [ ] Since this represents the main title of the `about.html` page, it would be nice to use an `h1` tag for semantic reasons and also it will help the search engines to know what the page is about. Therefore I suggest you change this `h3` to `h1` as suggested by the previous reviewer